src/mpid/ch4/shm/xpmem/xpmem_win.c

```diff
@@ -4,7 +4,7 @@
  *      See COPYRIGHT in top-level directory.
  */
 
-#include "xpmem_impl.h"
+#include "mpidimpl.h"
 #include "xpmem_seg.h"
```

Comment:
Is this change related to coop copy?

Reply:
I need check which commit this one belong to. No, the PR moved on and added more refactoring stuff. But please don't be alarmed and there is no pressure to merge any of these change if there is missing agreement. I have been wanting to see xpmem as a simple am protocol rather than as a "pseudo"-shmmod for a while, and this is my trying it out and soliciting discussions. If it works, ofi/ucx all can use the similar model adding lmt support or different variations of lmt support with relative ease.